test/built-ins/Object/assign/Target-Boolean.js

```diff
@@ -0,0 +1,12 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: Test the first argument(target) of Object.Assign(target,...sources),
+             if target is Boolean,the return value should be a new object whose value is target.			 
```

Comment:
This line (and a few others) has trailing whitespace that interferes with Test262's custom YAML parser. These tests shouldn't be merged until the whitespace is removed, but maybe it's straightforward enough that @bterlson can do it on @yanlxu's behalf.